optaplanner-examples/src/test/java/org/optaplanner/examples/tsp/persistence/LowEntityCountTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.examples.tsp.persistence;
```

Comment:
The package is not accurate. This test is not about TSP. It's an integration test that requires a real-world domain but it could work with any other (chained?) example. I suggest to move the test into `org.optaplanner.examples.integration_tests.lowentitycount` package. All integration tests that are being migrated from QE test suite should be placed under a dedicated package, e.g. `org.optaplanner.examples.integration_tests`. By contrast, packages of the individual examples (e.g. `org.optaplanner.examples.tsp`) will always only contain test that are intended to prove that the example works.

Let's talk if you disagree or if you have other suggestions.

Reply:
I fully agree with you. In QE suite test was placed in separate package `...examples.entitylowcount`, so we might just keep the structure?

Reply:
I would even add one extra level, `...examples.integrationtests.entitylowcount` so that all integration tests from QE test suite are clearly separated from examples tests.

Reply:
The tsp app package "org.optaplanner.examples.tsp.app" contains a lot of integration tests already. I wouldn't mind seeing it there.

Reply:
In this specific case, I'd move the test to a package that isn't associated with any specific example to make it clear the test focuses on a generic solver feature that is independent of any specific domain.

Reply:
Existing examples tests are pretty much standardized across all the examples (turtle tests, performance tests, benchmark tests). Although each QE test depends on some particular example, there is no such standardization. Introducing a new package like "integrationtests" would help distinguishing general integration tests and standard examples tests.